docs/common-patterns.rst

```diff
@@ -62,19 +66,23 @@ This is as opposed to the more intuitive sending pattern:
 ::
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.7.0 <0.9.0;
+    pragma solidity ^0.8.3;
```

Comment:
Yeah. I thought about pointing it out myself but I already complained about them in 0.8.1 and 0.8.2 so I thought it's better to wait until this PR is accepted instead of pushing for an update that may need another update in a moment :)

Anyway, the doc check should catch them.